torntools/scripts/content/items/ttItems.js

```diff
@@ -757,3 +758,22 @@ function showMissingFlowers() {
 		}
 	}
 }
+
+function showECanGains() {
+	let facECanPerc = parseInt(userdata.faction_perks.filter((x) => /energy drinks/i.test(x)).map((x) => {
+		x.replace(/[^0-9\.]/g, "");
+	})[0]);
```

Comment:
Could you add a comment in code what `/energy drinks/i.test(x)` and `x.replace(/[^0-9\.]/g, "")` do. Just for future reference as I don't know what the first one does and I don't want to crack my head in reading regex 😄 .